src/MainWindow.cpp

```diff
@@ -140,6 +149,19 @@ class MainWindowPrivate
         int numImages;
 };
 
+namespace {
+
+void SetOptionValue(RendererOptions& options,
+		RendererOptions::RenderOption option, bool on) {
+    if (on) {
+	options.options |= option;
+    } else {
+	options.options &= ~option;
+    }
+}
+
+}  // namespace
+
```

Comment:
The SetOptionValue is a replacement for this construct, which is copy-pasted in several places:
```if (M_PREFS->getBackgroundVisible()) p->renderOptions.options |= RendererOptions::BackgroundVisible; else p->renderOptions.options &= ~RendererOptions::BackgroundVisible;```

Becomes:
```SetOptionValue(p->renderOptions, RendererOptions::BackgroundVisible, M_PREFS->getBackgroundVisible());```